src/attributes.jl

```diff
@@ -910,6 +910,24 @@ A model attribute for the number of results available.
 """
 struct ResultCount <: AbstractModelAttribute end
 
+"""
+    ConflictStatus(N)
+    ConflictStatus()
+
+A model attribute for the `TerminationStatusCode` explaining why the conflict
+refiner stopped when computing the conflict `N`.
+
+If a minimal conflict is found, it will return `OPTIMAL`.
+If the problem is feasible, it will return `INFEASIBLE`.
+If `compute_conflict` has not been called yet, it will return
+`OPTIMIZE_NOT_CALLED`.
+"""
+struct ConflictStatus <: AbstractModelAttribute
+    N::Int
```

Comment:
we are moving towards result_index instead of N: https://github.com/JuliaOpt/MathOptInterface.jl/pull/999